### DISCUSSION TITLE
Error appears on startup

### DISCUSSION THREAD

**Discussion by bunny47591** [fluttercandies/flutter_photo_manager#862] [General]
An error occurs when starting the Android application. (version 2.5.0)

`src/main/kotlin/com/fluttercandies/photo_manager/core/PhotoManager.kt: (33, 84): Unresolved reference: DBUtils`

**Comment by CaiJingLong** [comment]
The CI system usually ensures that there are no compile-time errors. If such errors do occur, you should first run `flutter clean` first.
